src/openrct2/CmdlineSprite.cpp

```diff
@@ -53,6 +53,7 @@ struct SpriteFile
     rct_sprite_file_header Header;
     std::vector<rct_g1_element> Entries;
     std::vector<uint8_t> Data;
+    bool isAbsolute = false;
```

Comment:
(On commit https://github.com/OpenRCT2/OpenRCT2/pull/13590/commits/48b3fe8944c9ece0036c71c037941a7fbc2ada17)

This should probably be private, as it's only needed inside the struct (which might become a class now).
That's probably true for the other member variables at this point too

Reply:
Made private in subsequent commit